### DISCUSSION TITLE
Indicate when a Runner reached OOM / runner performance analytics

### DISCUSSION THREAD

**Discussion by Plasma** [community/community#62336] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi team,

I had CodeQL running on standard GH Actions runners and ultimately it was failing intermittently due to an out-of-memory issue (fixed with larger GH runner with higher memory limit), it showed up as a SIGSEV error (ultimately killed by OOM) during the Action's code compilation step.

It would be helpful if the performance of the runner or events like that were indicated in failed runs to better triage issues.

**Comment by jketema** [comment]
Hi @Plasma,

Thanks for your suggestion. We are actually working on surfacing these errors more clearly and providing actionable recommendations along with the errors.
